config.json

```diff
@@ -1,792 +1,791 @@
 {
   "azure_mgmt_analysis_services": {
     "azure_mgmt_analysis_services_2017_07_14": {
-      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/7aa3a5247895ba34d6cfec73e036bb66dc907d20/specification/analysisservices/resource-manager/readme.md",
+      "markdown": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/4a97039db7d2ca3acd5a11fe32d815864bdfc27d/specification/analysisservices/resource-manager/readme.md",
```

Comment:
are you just updating to the latest commit hash at the time you made this change?